backend/src/main/routes/CallRoute.ts

```diff
@@ -0,0 +1,165 @@
+import express from 'express';
+import { CommonRoutesConfig } from './CommonRoutesConfig';
+import { StatusCodes } from 'http-status-codes';
+import { inject, injectable } from 'tsyringe';
+import { CallService } from '../services/CallService';
+import { checkJwt, checkRole } from '../middleware/JWTMiddleware';
+import { Roles } from '../security/Roles';
+
+@injectable()
+export default class CallRoute extends CommonRoutesConfig {
+  constructor(@inject('express-app') app: express.Application, private callService: CallService) {
+    super(app, 'AccountRoute');
+  }
+
+  configureRoutes(): express.Application {
+    this.getApp()
```

Comment:
I think you can combine the routes for this file under call and calls. No need to call this.get() everytime

Reply:
You can only have one route for every this.getApp()